superset/viz.py

```diff
@@ -2062,6 +2062,40 @@ def get_data(self, df):
         }
 
 
+class DeckPolygon(BaseDeckGLViz):
+
+    """deck.gl's Polygon Layer"""
+
+    viz_type = 'deck_polygon'
+    verbose_name = _('Deck.gl - Polygon')
+    deser_map = {
+        'json': json.loads,
+        'polyline': polyline.decode,
+    }
+
+    def get_position(self, d):
+        return d[self.form_data.get('line_column')]
+
+    def query_obj(self):
+        d = super(DeckPolygon, self).query_obj()
+        line_col = self.form_data.get('line_column')
+        if d['metrics']:
+            d['groupby'].append(line_col)
+        else:
+            d['columns'].append(line_col)
+        return d
+
+    def get_properties(self, d):
+        fd = self.form_data
+        deser = self.deser_map[fd.get('line_type')]
+        polygon = deser(d[fd.get('line_column')])
+        if fd.get('reverse_long_lat'):
+            polygon = (polygon[1], polygon[0])
+        return {
+            'polygon': polygon,
```

Comment:
Since this is pretty much the only difference between DeckPath and DeckPolyline, I'd suggest have `DeckPolyline` derive `DeckPath`, and renaming this key to something generic like `points`. If we wanted to have a different key on each context, you could do it through a class (static) attribute that would be different for each class.